### PR TITLE
New Package: YairAichenbaum.Files version 2.4.4.0

### LINKED ISSUE #91790: [Package Request]: Files
### Package Requested

- [X] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [X] I have performed a search and couldn't find this package.
- [X] I have checked that the installer for this package is not contained in a .zip file
- [ ] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Files Community
Publisher Url: https://github.com/files-community/Files
Package Name: Files
Description: Explore a beautiful Windows-first design. Manage all your files with increased productivity. Work across multiple folders with tabs. And so much more.
Package Version: v2.4.3
Installer URL: https://github.com/files-community/Files/releases/download/v2.4.3/Files.Package_2.4.3.0_x64_ARM64_Sideload.msixbundle

Current Package Identifier (for new versions):

### PR DESCRIPTION
**Pull request by mdanish-kh**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
- Closes #91790

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91910)

### CODE REVIEW

**Comment by mdanish-kh** [comment]
smh...I used the PackageFamilyName of previous version that was on preview..gotta use wingetcreate for msix I guess 😄

**Comment by Trenly** [comment]
> Installer failed with exit code: 0x80073cf3 : Package failed updates, dependency or conflict validation.
This package has a dependency missing from your system.

**Comment by stephengillie** [comment]
Adding `9P7VS8XHG9G5` as a dependency generated this log error:

```
2022-12-22 10:21:22.457 [CLI ] Manifest fields: Name [WindowsAppRuntime.1.2], Version [Unknown]
2022-12-22 10:21:22.491 [CLI ] Get user entitlement.
2022-12-22 10:21:22.497 [CLI ] Get device entitlement.
2022-12-22 10:21:22.854 [CLI ] Get entitlement succeeded. Server error. ProductId: 9P7VS8XHG9G5
2022-12-22 10:21:22.854 [CLI ] Terminating context: 0x8a15001e at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\MSStoreInstallerHandler.cpp:7a
2022-12-22 10:21:22.854 [CLI ] Terminating context: 0x8a150110 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:231
```

**Comment by Trenly** [comment]
The publisher should probably use the zip redistributable and package the msix files with their application - https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads

File: manifests/y/YairAichenbaum/Files/2.4.4.0/YairAichenbaum.Files.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: YairAichenbaum.Files
+PackageVersion: 2.4.4.0
+ReleaseDate: 2022-12-20
+Installers:
```
Comment (stephengillie) [inline manifests/y/YairAichenbaum/Files/2.4.4.0/YairAichenbaum.Files.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.WindowsAppRuntime.1.2
Installers:
```

Comment (stephengillie) [reply manifests/y/YairAichenbaum/Files/2.4.4.0/YairAichenbaum.Files.installer.yaml:7]: I've tested the above dependency in a VM, and it installs and runs successfully.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**